src/Control/Monad/Ghc.hs

```diff
@@ -14,6 +14,9 @@ import Control.Monad.Catch
 import Data.IORef
 
 import qualified GHC
+#if MIN_VERSION_ghc(9,2,0)
+import qualified GHC.Utils.Logger as GHC
+#endif
```

Comment:
if those ifdefs become too confusing, I'm willing to drop support for older versions of ghc, say, older than 8.10.

Reply:
Debian stable, testing and unstable is on ghc 8.4.4. https://packages.debian.org/search?keywords=ghc

Reply:
I see!

Reply:
err, I actually dropped support for ghc 8.4 [two releases ago](https://github.com/haskell-hint/hint/blob/main/CHANGELOG.md#0903)! I am guessing that Debian is not just using an older compiler, but also older libraries? Or is the fact that hint does not support ghc-8.4 currently causing problems in the Debian community? I'm happy to increase the range of supported versions if requested, I just default to a shorter support window in order to avoid wasting effort in the likely case that nobody cares.

Reply:
As no-one has complained, I guess it's not an issue. I think a fair number of Tidal users are on 8.4, but hint isn't a dependency for them.